### ISSUE TITLE
Cannot run any toolbox command after update

### ISSUE THREAD

**Issue by jgresty** [containers/toolbox#477]
`toolbox list -v`
```
DEBU Running as real user ID 1000                 
DEBU Resolved absolute path to the executable as /usr/bin/toolbox 
DEBU Running on a cgroups v1 host                 
DEBU Checking if /etc/subgid and /etc/subuid have entries for user user 
DEBU TOOLBOX_PATH is /usr/bin/toolbox             
DEBU Toolbox config directory is /home/user/.config/toolbox 
DEBU Current Podman version is 2.0.0              
DEBU Old Podman version is 2.0.0                  
DEBU Migration not needed: Podman version 2.0.0 is unchanged 
DEBU Fetching images with label=com.redhat.component=fedora-toolbox 
DEBU Fetching images with label=com.github.debarshiray.toolbox=true 
panic: interface conversion: interface {} is nil, not string

goroutine 1 [running]:
github.com/containers/toolbox/pkg/utils.JoinJSON(0x55c010a700d9, 0x2, 0xc0000efcb8, 0x2, 0x2, 0x4, 0x0, 0x0)
	github.com/containers/toolbox/pkg/utils/utils.go:477 +0x3a7
github.com/containers/toolbox/cmd.listImages(0x55c010a74ab4, 0x12, 0x55c010b07600, 0xc0000989e0, 0x55c010b06c40)
	github.com/containers/toolbox/cmd/list.go:165 +0x2dc
github.com/containers/toolbox/cmd.list(0x55c010d49120, 0xc000098910, 0x0, 0x1, 0x0, 0x0)
	github.com/containers/toolbox/cmd/list.go:90 +0x12f
github.com/spf13/cobra.(*Command).execute(0x55c010d49120, 0xc0000988f0, 0x1, 0x1, 0x55c010d49120, 0xc0000988f0)
	github.com/spf13/cobra@v0.0.6/command.go:840 +0x455
github.com/spf13/cobra.(*Command).ExecuteC(0x55c010d49ba0, 0x0, 0xc0000eff48, 0x55c01082469c)
	github.com/spf13/cobra@v0.0.6/command.go:945 +0x319
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v0.0.6/command.go:885
github.com/containers/toolbox/cmd.Execute()
	github.com/containers/toolbox/cmd/root.go:70 +0xb8
main.main()
	github.com/containers/toolbox/toolbox.go:22 +0x2
```

Trying to enter a toolbox that already exists: `toolbox enter -c default -v`
```
DEBU Running as real user ID 1000                 
DEBU Resolved absolute path to the executable as /usr/bin/toolbox 
DEBU Running on a cgroups v1 host                 
DEBU Checking if /etc/subgid and /etc/subuid have entries for user user 
DEBU TOOLBOX_PATH is /usr/bin/toolbox             
DEBU Toolbox config directory is /home/user/.config/toolbox 
DEBU Current Podman version is 2.0.0              
DEBU Old Podman version is 2.0.0                  
DEBU Migration not needed: Podman version 2.0.0 is unchanged 
DEBU Resolving container and image names          
DEBU Container: 'default'                          
DEBU Image: ''                                    
DEBU Release: ''                                  
DEBU Resolved container and image names           
DEBU Container: 'default'                          
DEBU Image: 'fedora-toolbox:30'                   
DEBU Release: '30'                                
DEBU Checking if container default exists          
DEBU Calling org.freedesktop.Flatpak.SessionHelper.RequestSession 
DEBU Starting container default                    
Error: failed to start container default
```

`toolbox create -c foobar -r 31 -v `
```
DEBU Running as real user ID 1000                 
DEBU Resolved absolute path to the executable as /usr/bin/toolbox 
DEBU Running on a cgroups v1 host                 
DEBU Checking if /etc/subgid and /etc/subuid have entries for user user 
DEBU TOOLBOX_PATH is /usr/bin/toolbox             
DEBU Toolbox config directory is /home/user/.config/toolbox 
DEBU Current Podman version is 2.0.0              
DEBU Old Podman version is 2.0.0                  
DEBU Migration not needed: Podman version 2.0.0 is unchanged 
DEBU Resolving container and image names          
DEBU Container: 'foobar'                          
DEBU Image: ''                                    
DEBU Release: '31'                                
DEBU Resolved container and image names           
DEBU Container: 'foobar'                          
DEBU Image: 'fedora-toolbox:31'                   
DEBU Release: '31'                                
DEBU Checking if container foobar already exists  
DEBU Looking for image fedora-toolbox:31          
DEBU Resolving fully qualified name for image fedora-toolbox:31 
DEBU Resolved image fedora-toolbox:31 to registry.fedoraproject.org/f31/fedora-toolbox:31 
DEBU Looking up group for sudo                    
DEBU Group for sudo is wheel                      
DEBU Checking if 'podman create' supports '--ulimit host' 
DEBU 'podman create' supports '--ulimit host'     
DEBU Resolving path to the D-Bus system socket    
DEBU Calling org.freedesktop.Flatpak.SessionHelper.RequestSession 
DEBU /home/user canonicalized to /home/user 
DEBU Checking if /usr is mounted read-only or read-write 
DEBU Mount-point of /usr is /                     
DEBU Mount flags of /usr on the host are rw,relatime,discard 
DEBU Resolving path to the KCM socket             
DEBU failed to find a SOCK_STREAM socket for sssd-kcm.socket 
DEBU Checking if /mnt is a symbolic link to /var/mnt 
DEBU Looking for toolbox.sh                       
DEBU Found /etc/profile.d/toolbox.sh              
DEBU Checking if /home is a symbolic link to /var/home 
DEBU Creating container foobar:                   
DEBU podman                                       
DEBU --log-level                                  
DEBU error                                        
DEBU create                                       
DEBU --dns                                        
DEBU none                                         
DEBU --env                                        
DEBU TOOLBOX_PATH=/usr/bin/toolbox                
DEBU --group-add                                  
DEBU wheel                                        
DEBU --hostname                                   
DEBU toolbox                                      
DEBU --ipc                                        
DEBU host                                         
DEBU --label                                      
DEBU com.github.containers.toolbox=true           
DEBU --label                                      
DEBU com.github.debarshiray.toolbox=true          
DEBU --name                                       
DEBU foobar                                       
DEBU --network                                    
DEBU host                                         
DEBU --no-hosts                                   
DEBU --pid                                        
DEBU host                                         
DEBU --privileged                                 
DEBU --security-opt                               
DEBU label=disable                                
DEBU --ulimit                                     
DEBU host                                         
DEBU --userns=keep-id                             
DEBU --user                                       
DEBU root:root                                    
DEBU --volume                                     
DEBU /etc:/run/host/etc                           
DEBU --volume                                     
DEBU /dev:/dev:rslave                             
DEBU --volume                                     
DEBU /run:/run/host/run:rslave                    
DEBU --volume                                     
DEBU /tmp:/run/host/tmp:rslave                    
DEBU --volume                                     
DEBU /var:/run/host/var:rslave                    
DEBU --volume                                     
DEBU /run/dbus/system_bus_socket:/run/dbus/system_bus_socket 
DEBU --volume                                     
DEBU /run/user/1000/.flatpak-helper/monitor:/run/host/monitor 
DEBU --volume                                     
DEBU /home/darkelink:/home/user:rslave       
DEBU --volume                                     
DEBU /usr/bin/toolbox:/usr/bin/toolbox:ro         
DEBU --volume                                     
DEBU /usr:/run/host/usr:rw,rslave                 
DEBU --volume                                     
DEBU /run/user/1000:/run/user/1000                
DEBU --volume                                     
DEBU /mnt:/mnt:rslave                             
DEBU --volume                                     
DEBU /run/media:/run/media:rslave                 
DEBU --volume                                     
DEBU /etc/profile.d/toolbox.sh:/etc/profile.d/toolbox.sh:ro 
DEBU registry.fedoraproject.org/f31/fedora-toolbox:31 
DEBU toolbox                                      
DEBU --verbose                                    
DEBU init-container                               
DEBU --home                                       
DEBU /home/user                              
DEBU --monitor-host                               
DEBU --shell                                      
DEBU /bin/zsh                                     
DEBU --uid                                        
DEBU 1000                                         
DEBU --user                                       
DEBU user                                    
Error: invalid config provided: Groups and privileged are mutually exclusive options
Error: failed to create container foobar
```

**Comment by HarryMichal** [comment]
Hi @jgresty! We're aware of these issues with Podman 2.0.0. I already have a WIP pull request that fixes some of those errors #471. Some more PRs are pending to be tested before they're included in 'master' and new release is made.

**Comment by HarryMichal** [comment]
Toolbox [v0.0.91](https://github.com/containers/toolbox/releases/tag/0.0.91) should have all the needed fixes to work with Podman V2. In Podman were some bugs also fixed.
